meta-balena-common/recipes-core/base-files/base-files/nsswitch.conf

```diff
@@ -8,7 +8,7 @@ passwd:         compat ato
 group:          compat
 shadow:         compat
 
-hosts:          files dns
+hosts:          files myhostname dns mdns [NOTFOUND=return]
```

Comment:
@jakogut we may need to do some tuning of this in IPv4 only vs. IPv4/IPv6 networks. Also need to test in IPv6 only...

Reply:
I've been using this on my test devices without any problems (I have IPv6):

```sh
# Host OS
$ mount -o remount r,w /

$ sed -i 's/hosts:.*/hosts:\t\tfiles myhostname dns mdns4 mdns \[NOTFOUND=return\]/g' \
    /etc/nsswitch.conf

$ cat > /etc/mdns.allow << EOF
.local.
.local
EOF
```